vision/src/Classification.kt

```diff
@@ -11,21 +11,20 @@
 //  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
-package com.google.kotlinvision;
-
-import com.google.cloud.vision.v1.AnnotateImageRequest;
-import com.google.cloud.vision.v1.Feature;
-import com.google.cloud.vision.v1.Feature.Type;
-import com.google.cloud.vision.v1.Image;
-import com.google.cloud.vision.v1.ImageAnnotatorClient;
-import com.google.protobuf.ByteString;
+package com.google.kotlinvision
+import com.google.cloud.vision.v1.AnnotateImageRequest
+import com.google.cloud.vision.v1.Feature
+import com.google.cloud.vision.v1.Feature.Type
+import com.google.cloud.vision.v1.Image
+import com.google.cloud.vision.v1.ImageAnnotatorClient
+import com.google.protobuf.ByteString
 
 import java.io.IOException
 import java.io.File
 
 fun main(args: Array<String>) {
-    var requests = ArrayList<AnnotateImageRequest>()
-    val imageFile = if (args.size == 0) {
+    val requests = ArrayList<AnnotateImageRequest>()
```

Comment:
What is the reasoning behind changing this to `val`? Because this is being added to (line 48) I believe this is more appropriately `var`.

Reply:
Okay, at line 48 we are editing the object (ArrayList) the variable "requests" references, not the variable itself. Hope that makes some sense.